charts/osm/values.yaml

```diff
@@ -55,6 +55,7 @@ OpenServiceMesh:
   controllerLogLevel: trace
   enforceSingleMesh: false
   webhookConfigNamePrefix: osm-webhook
+  osmNamespace: ""
```

Comment:
Can we add a comment here and in the chart readme that if `osmNamespace` is not provided, then the release namespace is used as the namespace for where the osm controller will be installed?